README.md

```diff
@@ -153,6 +153,7 @@ The `views` variable should be provided as a list of object with the following k
       billable = "true"
   }
   description = "A long description"                                 # Text describing the contents or purpose of the view.
+  schema = "<SCHEMA JSON DATA>"                                      # (Optional) JSON schema, to allow setting options such as field description
 }
 ```
 
```

Comment:
Also would be nice to update the [inputs](https://github.com/cbsi-dto/tf-mod-bq/blob/d0a4b7c804972a3bf48c803dca2538f00465c9de/README.md#inputs) from the readme.
https://github.com/cbsi-dto/tf-mod-bq/blob/f3afc9b296726cb9c5f4aa08808ea9a89df70987/README.md?plain=1#L201

Reply:
Updated this part of the docs